oam/install_scripts/mcs-exemgr.service.in

```diff
@@ -5,7 +5,7 @@ After=mcs-writeengineserver.service
 
 [Service]
 Type=simple
-Environment="SKIP_OAM_INIT=1"
+Environment="LD_PRELOAD=libjemalloc.so.1 libjemalloc.so.2"
```

Comment:
There is good news; there is bad news. LD_PRELOAD will look for shared libs in the standard lib paths (i.e., /usr/lib64, /usr/lib, /usr/lib/x86_64-linux-gnu) so we do not actually need a direct path to libjemalloc.so. The issue we have is we won't be sure which version (so.1, so.2) user will have until runtime. Setting this environment variable requires an exact name, and not a dynamic variable, so the quick solution is to cut our loss (the one non-terminating error msg in journalctl for the lib that doesn't exist). I had a separate thought, where we determine which version of libjemalloc we have during cmake, however we cannot be certain that user has libjemalloc installed at that time, which can introduce the case where LD_PRELOAD="" and then we never load libjemalloc.

Reply:
can we say that regular libjemalloc.so (no version) always exists?

Reply:
no, looks like the non-versioned name doesn't exist on deb systems, at least not my mint 19 box.  pita.  Might have to get fancy and find which one to use at runtime.  For reference, I had to add code like that to the old 'columnstore' script.

Reply:
I have modified this to not get an error and always choose correct .so